docs/admin-portal.md

```diff
@@ -0,0 +1,95 @@
+# Admin Portal
+
+## Introduction
+
+The admin portal is a SRE-facing front end used for performing various actions and retrieving information for clusters in a given region.
+
+The admin portal runs inside the RP and calls against the RP for cluster information and serves that information using a MSFT inspired front end.
+
+The front end is developed using react and typescript. The back end api is written in golang and makes direct calls to CosmoDB for cluster information.
+
+The portal front end lives in the top level directory of the ARO-RP repo within the `portal` directory. The portal back end exists within the `pkg/portal`
+
+The front end code is compiled into go code using the bindata golang module. This front end code is then served through the RP.
+
+
+## Developing
+
+You will require Node.js and `npm`. These instructions were tested with the versions from the Fedora 34 repos.
+
+1. Make your desired changes in `portal/v2/src/` and commit them.
+
+1. Run `make build-portal` from the main directory. This will install the dependencies and kick off the Webpack build, placing the results in `portal/v2/build/`.
+
+1. Run `make generate`. This will regenerate the golang file containing the portal content to be served.
+
+1. Commit the results of `build-portal` and `generate`.
+
+## Running Admin Portal in development
+
+### Running Portal Served from development RP
+
+1. Complete Steps mentioned above to build and compile portal.
+
+1. Make sure development environment variables are set and also set `export NO_NPM=1`
+
```

Comment:
So what it does is makes sure the RP points to localhost:8444 instead of redirecting to localhost:3000 which is currently the default behavior in development. But I have found there are times you just want to test the RP as it would appear in production so I felt this flag was necessary. I will add it to the documentation

Reply:
Updated